wayland/install-wayland.txt

```diff
@@ -0,0 +1,158 @@
+Wayland
+________________________________________________________________________________
+
+Wayland [0] is a display protocol that aims to be a simpler replacement for the
+X Window System. [1] Unlike Xorg, which provides a unified, featureful base for
+the interchangeable window manager, the Wayland protocol as is, defines merely a
+way for the GUI to talk to the underlying graphics interface. This protocol has
+to be implemented in a library [2][3][4] first, which in return provides the
+necessary Wayland extensions to create a compositor [5][6][7] upon. There is no
+unified way on how to implement Wayland, so the feature set between compositors
+may vary, but they typically do the heavy lifting from grabbing input events to
+allocating system resources and arrange all that for the user in a usable GUI.
+
+
+Caveats
+________________________________________________________________________________
+
+Aside from the fact that Wayland and it's implementations are in an early state
+of evolvement and Xorg still defines the GUI standard regarding stability,
+compatibility and portability, the biggest downside is choice!
+The different Wayland implementations are followed by lock ins towards supported
+hardware, device management and software for certain use cases which might not
+be portable across other implementations. Not all software needs may be
+satisfied yet.
+
+NOTE: This list is rather incomplete and might change, but it gives an idea on
+      what to expect.
+
+- Limited hardware and driver support. [8] [9]
+- Fragmentation accross the different Wayland implementations lead to special, 
+  non portable software for bar support, screen sharing, screen recording, ...
+  [10] [11] [12]
+- To extend compatibility it is possible to start applications with XWayland, an
```

Comment:
This line is one character too long.

Reply:
`- To extend compatibility it is possible to start applications with XWayland, an` is exactly 80 columns wide.

Reply:
My bad.